<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogProva</class>
 <widget class="QDialog" name="DialogProva">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnCrear">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear bola</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Poscion X:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePosX">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Poscion Y:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePosY">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Velocidad:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leVel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Apariencia:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbApariencia">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Radio:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leRadio">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnElegir">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Elegir</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
